src/en/developer-leadership.md

```diff
@@ -98,7 +98,9 @@ on users of `leader-set`.
 
 ### How do I...
 
-### ...share one-shot configuration among units?
+Click on the expander to see details for each.
+
+^# ...share one-shot configuration among units?
 
 Assuming your own implementations of `create_settings` and `valid_settings`, you
```

Comment:
It renders okay locally, probably because of the different css style when using the foldouts makes them look subordinate. Want these changed to `^###` ? I'm happy to do so.